specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/connectors.json

```diff
@@ -0,0 +1,909 @@
+            "required": true,
+            "type": "string",
+            "description": "The name of the workbench"
+          },
+          {
+            "name": "chamberName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the chamber."
+          },
+          {
+            "in": "path",
+            "name": "name",
+            "required": true,
+            "type": "string",
+            "description": "The name of the connector."
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "description": "License key parameters",
+            "schema": {
+              "$ref": "#/definitions/LicenseKeyParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default Error Response",
+            "schema": {
+              "$ref": "./modsimworkbenches.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Connectors License Add": {
+            "$ref": "./examples/ConnectorsLicenseAdd.json"
+          }
+        },
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "RemoteDesktop": {
+      "type": "object",
+      "description": "Remote Desktop properties.",
+      "properties": {
+        "etx": {
+          "$ref": "#/definitions/Etx",
+          "description": "The etx object."
+        }
+      }
+    },
+    "Etx": {
+      "type": "object",
+      "properties": {
+        "connectionNodeCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The connection node count."
+        },
+        "licenseActivated": {
+          "description": "The license activated.",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "licenseActivated",
+            "modelAsString": true
+          }
+        },
+        "replyUrlEtx": {
+          "description": "The reply URL of ETX Server for Azure AD app.",
+          "type": "string",
+          "readOnly": true
+        },
+        "replyUrlOtds": {
+          "description": "The reply URL of OTDS for Azure AD app.",
```

Comment:
>OTDS [](http://example.com/codeflow?start=43&length=4)

Please expand out less-well-known acronyms.

Reply:
e.g. "The reply URL of the one-time distributed store (OTDS) for Azure AD app." 
-or whatever it really stands for